src/map/attack.cpp

```diff
@@ -445,7 +445,7 @@ void CAttack::ProcessDamage(bool isCritical, bool isGuarded, bool isKick)
 {
     // Sneak attack.
     if (m_attacker->GetMJob() == JOB_THF && m_isFirstSwing && m_attacker->StatusEffectContainer->HasStatusEffect(EFFECT_SNEAK_ATTACK) &&
-        ((abs(m_victim->loc.p.rotation - m_attacker->loc.p.rotation) < 23) || m_attacker->StatusEffectContainer->HasStatusEffect(EFFECT_HIDE) ||
+        (behind(m_attacker->loc.p, m_victim->loc.p, 64) || m_attacker->StatusEffectContainer->HasStatusEffect(EFFECT_HIDE) ||
```

Comment:
Might want to have this be held by a private variable once evaled for position in some form of unified function.  We do a LOT of position checks in this code, to when we should only do until true, and then keep that state in the future.